Packages/Status/Sources/Status/Editor/Components/StatusEditorAccessoryView.swift

```diff
@@ -14,6 +14,8 @@ struct StatusEditorAccessoryView: View {
   @FocusState<Bool>.Binding var isSpoilerTextFocused: Bool
   @ObservedObject var viewModel: StatusEditorViewModel
   @State private var isDrafsSheetDisplayed: Bool = false
+
+  @Binding var isShowingPoll: Bool
```

Comment:
Why do we need a binding var here instead of just using the existing `viewModel.showPoll` flag?

Reply:
Nice catch! 😿 Yes, you are right. Unfortunately, I moved my POC code into this code base, and it appears I forgot to change this to use the view model.